### ISSUE TITLE
[simulation] release notes not up to date (Trac #1325)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1317]
The simulation meta-project trunk release notes only goes up to V04-01-07.  I've seen release notes for versions past that, so we're out of sync. Fix that.

Migrated from https://code.icecube.wisc.edu/ticket/1325
```json
{
    "status": "closed", 
    "changetime": "2016-03-18T21:14:03", 
    "description": "The simulation meta-project trunk release notes only goes up to V04-01-07.  I've seen release notes for versions past that, so we're out of sync. Fix that.", 
    "reporter": "david.schultz", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1458335643235016", 
    "component": "combo simulation", 
    "summary": "[simulation] release notes not up to date", 
    "priority": "blocker", 
    "keywords": "", 
    "time": "2015-09-01T22:48:13", 
    "milestone": "", 
    "owner": "olivas", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150901T22:57:38: olivas commented:

> https://wiki.icecube.wisc.edu/index.php/IceSim_Releases
> 
> I'll get those integrated into the notes in the repo, since we're moving away from wiki for these kinds of things, but there's the link if you want a reference in the meantime.

**Comment by icecube-trac** [comment]
Trac update at 20150901T23:19:30: 
* olivas changed status from "new" to "closed"
* olivas changed resolution from "" to "fixed"
* olivas commented:

> In changeset 137155:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="137155"
> merged release notes from the wiki.  closes #1317.
> ```

**Comment by icecube-trac** [comment]
Trac update at 20160318T21:14:03: 
* olivas changed milestone from "IceSim 4.2" to ""
* olivas commented:

> Milestone IceSim 4.2 deleted
